pkgs/applications/virtualization/ovmfvartool/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenvNoCC, fetchFromGitHub, python3 }:
+
+let
+  pythonPkg = python3.withPackages (p: with p; [ pyyaml ]);
+in stdenvNoCC.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```
Can you move `pythonPkg` to `postPatch` as it is only used there?